external/fv3fit/fv3fit/_shared/input_sensitivity.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+from typing import Optional, Mapping
+from dataclasses import dataclass
+
+
+RandomForestInputSensitivity = Mapping[str, Mapping[str, np.ndarray]]
```

Comment:
Ideally I wanted to make these dataclasses that would be constructed in the RF/NN `input_sensitivity` methods but couldn't find a way to define these in the individual model's modules without running into circular imports when trying to import here for type hinting. If you have a better suggestion let me know.

Reply:
I don't have any solutions to the circular import issue, could you still use dataclasses to better specify the `{method}InputSensitivity` data structures? Right now the jacobian and RF importance ones look identical from the typing alone, but of course they have pretty different data meanings. It would help make it more obvious what the data are/how they should be plotted.

Reply:
Sure, I added one for the RF input sensitivity. I couldn't think of a good way to use a data struct for the Jacobian data as it's a dict of dicts, which gets kind of unwieldy and harder to traverse later if I try to make a formal data structure out of it.